openspec/specs/batch-notification-aggregation/spec.md

```
@@ -1,41 +1,47 @@
-- **THEN** 系统展示一条聚合 toast 通知："新增了 2 个线索：凶器、日记"
+#### Scenario: 批量新增道具
+
+- **WHEN** Agent 批量新增 2 个道具（凶器、日记），SSE 变更批次包含 2 条 `prop:created` 变更
+- **THEN** 系统展示一条聚合 toast 通知："新增了 2 个道具：凶器、日记"
 
 #### Scenario: 单条变更保持原有格式
+
 - **WHEN** SSE 变更批次仅包含 1 条 `character:created` 变更
 - **THEN** 通知文案保持与当前行为一致（如"角色「张三」已创建"）
 
 ### Requirement: 变更分组展示
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)